### ISSUE TITLE
Summary list row actions within summary cards must specify which summary card they're part of

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-frontend#3674]
This issue is from May 2023 external accessibility audit report.

**WCAG Reference:**
2.4.4 Link Purpose (In Context) (Level A)
[Understanding Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html) | [How to Meet Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/quickref/#link-purpose-in-context)

2.4.9 Link Purpose (Link Only) (Level AAA)
[Understanding Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html) | [How to Meet Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/quickref/#link-purpose-link-only)

**Issue ID:**
DAC_Link_Purpose_02

**URLs:**
[https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/work-history#/delete](https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/work-history#/delete)

## Screen shots
<img width="637" alt="Summary card component with 'change' link" src="https://github.com/alphagov/govuk-frontend/assets/56260216/2b31d608-754b-48e2-aa3a-0b3bb71276fc">

The ‘change’ links have been provided with additional link text to inform users of screen reading assistive technologies that they relate to the different questions i.e., ‘job title’. However, additional context is required to understand their whole purpose as to which section and work they relate to. For example, a user would need to navigate backwards to locate the heading to which they are sectioned under, and as there are multiple ‘change job title’ links, this can be confusing for users.

## Current code ref(s)
`#main-content > div > div > div:nth-child(3) > div.govuk-summary-card__content > dl > div:nth-child(1) > dd.govuk-summary-list__actions > a`

## Screen-reader comments
> “I located multiple links on the page which read to me as ‘Delete work history’. These links related to different sections of the page and would delete different areas although at the time of testing did not perform a function. I was not aware when browsing out of context that the links would perform a different action or what would be deleted as the links did not contain enough description within the link text to make this obvious. Giving each link text clear and unique information that can be understood either in or out of context will prevent any confusion and will ensure that I can select easily.”

## Solution
Ensure that all links are descriptive of their purpose and functionality, allowing users of screen reading assistive technologies to confidently interact and understand the link.
The best way to make sure of this, is to locate links where some additional link context is needed to understand the purpose of the link. For each link:
1. Check whether the context is contained in the same sentence, paragraph, list item, table cell, or associated table headers.
2. Check whether the link context can be programmatically determined in some other way, for example, by using a WAI-ARIA property such as aria-label, aria-labelledby or aria-describedby on the link to provide sufficient context.
Consider adding additional text including the heading or work section to give a clear purpose for the link.

**Comment by davidc-gds** [comment]
This issue has been assessed to not be a high-severity accessibility concern. It only meets 1 of our 3 [criteria for a high-severity accessibility concern](https://design-system.service.gov.uk/community/accessibility-strategy/#determining-severity-of-an-accessibility-concern).

- There is reasonable evidence that it makes it difficult or impossible for some people to complete certain tasks. ✅
The evidence:
    - It is related to a WCAG level A failure.
    - It was assessed in an external accessibility audit as a 'high' priority accessibility concern.
- It does not appear to affect the accessibility of essential services or critical infrastructure. ❌
- There is no blocker to individual service teams efficiently resolving it on their own. ❌

As such, this will not be flagged as a high-priority issue.

In the coming weeks we will investigate whether it fails WCAG level A criterion: [2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html).

**Comment by CharlotteDowns** [comment]
This has also already been captured in [Accessibility issue: duplicate links on example for summary cards](https://github.com/alphagov/govuk-design-system/issues/2680)
